http/user_resource_mapping_service.go

```diff
@@ -274,30 +273,16 @@ func decodeDeleteMemberRequest(ctx context.Context, r *http.Request) (*deleteMem
 
 // FindUserResourceMappings returns the user resource mappings
 func (s *UserResourceMappingService) FindUserResourceMappings(ctx context.Context, filter influxdb.UserResourceMappingFilter, opt ...influxdb.FindOptions) ([]*influxdb.UserResourceMapping, int, error) {
-	url, err := NewURL(s.Addr, s.basePath)
+	url, err := NewURL(s.Addr, resourceIDPath(filter.ResourceType, filter.ResourceID, string(filter.UserType)+"s"))
```

Comment:
the `+"s"` piece here is interesting, why is this needed here?

Reply:
i think it's just some old code. We could make another issue to clean them up